README.md

```diff
@@ -4,6 +4,113 @@
 
 This library provides instrumentation for CDI modules in Java EE applications.
 
+## Usage
+
+For a concrete and comprehensive set of examples, refer to the `opentracing-cdi-example`
+module.
+
+### Injecting a tracer
+
+In your managed beans, you can get an instance of the appropriate tracer via:
+
+```java
+@Inject
+Tracer tracer;
+```
+
+If you have your own `Tracer` producer, you'll need to mark your own as an
+`@Alternative`
```

Comment:
> About alternative etc. I think we should just use the simplest and most intuitive solution.

This all depends on what's the common case. The simplest and "most intuitive" solution is to just let people add stuff as maven dependency and consume a tracer via `@Inject Tracer tracer`. All boiler plate code, like creating a new tracer, would be done by "something" (TracerResolver, CDI integration, ...). It becomes a problem if the common case is for people to specify their own `Tracer`.

For my demos and examples, I never create a Tracer by myself: I always use system properties to configure Jaeger's Tracer, and use Jaeger's TracerResolver module, so that the tracer of choice becomes more like a packaging/runtime decision and less of a developer's decision.